src/promptflow-evals/promptflow/evals/evaluators/groundedness/__init__.py

```diff
@@ -28,18 +29,26 @@ def init(model_config: AzureOpenAIConnection):
             context="Tokyo is Japan's capital, known for its blend of traditional culture \
                 and technological advancements.")
     """
+
     def eval_fn(answer: str, context: str):
         # Load the flow as function
         current_dir = Path(__file__).resolve().parent
         flow_dir = current_dir / "flow"
         f = load_flow(source=flow_dir)
 
         # Override the connection
-        f.context.connections = { 
-            "query_llm": { "connection": model_config } 
+        f.context.connections = {
+            "query_llm": {"connection": AzureOpenAIConnection(
+                api_base=model_config.api_base,
+                api_key=model_config.api_key,
+                api_version=model_config.api_version,
+                api_type="azure"
+            ),
+                "deployment_name": model_config.deployment_name,
```

Comment:
Move this line one line up. It should be inside AzureOpenAIConnection(...). Pass `model_config.model` to the constructor as well.

Reply:
AzureOpenAIConnection does not have a model parameter.

Reply:
Ah, right. Sorry I am going too fast.

This should work:

```
        # Override the connection
        f.context.connections = { 
            "query_llm": { "connection": model_config },
        }
        f.context.overrides = {
            "nodes.query_llm.inputs.deployment_name": model_config.deployment_name,
        }
```

Reply:
What I am doing is exactly that just a slightly different syntax. These 2 lines are merged together in a way